src/main/java/emissary/core/constants/Configurations.java

```diff
@@ -0,0 +1,38 @@
+package emissary.core.constants;
+
+import com.google.common.collect.Lists;
+
+import java.util.Collections;
+import java.util.List;
+
+public class Configurations {
+
+    // common configs
+    public static final String END_FORM = "END_FORM";
+    public static final String NEW_FORM = "NEW_FORM";
+    public static final String OUTPUT_FORM = "OUTPUT_FORM";
+    public static final String PLACE_RESOURCE_LIMIT_MILLIS = "PLACE_RESOURCE_LIMIT_MILLIS";
+
+    // reserved configs for service/place creation
+    public static final String PLACE_NAME = "PLACE_NAME";
+    public static final String SERVICE_NAME = "SERVICE_NAME";
+    public static final String SERVICE_TYPE = "SERVICE_TYPE";
+    public static final String SERVICE_DESCRIPTION = "SERVICE_DESCRIPTION";
+    public static final String SERVICE_COST = "SERVICE_COST";
+    public static final String SERVICE_QUALITY = "SERVICE_QUALITY";
+    public static final String SERVICE_PROXY = "SERVICE_PROXY";
+    public static final String SERVICE_KEY = "SERVICE_KEY";
+
+    public static final List<String> RESERVED_CONFIGS_SERVICE = Collections.unmodifiableList(
```

Comment:
this would've been helpful for #352

Reply:
I was trying to think of a prefix for reserved configs, but didn't do a good job of naming - added the java doc and renamed it. Hopefully, this is a little better.